acceptance/acceptance_test.go

```diff
@@ -249,6 +261,15 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 	cmd.Dir = tmpDir
 	err = cmd.Run()
 
+	// Write the requests made to the server to a output file if the test is
+	// configured to record requests.
+	if config.RecordRequests {
+		requests := server.Requests()
+		requestsJSON, err := json.MarshalIndent(requests, "", "  ")
```

Comment:
Can we marshal each request independently using jsonlines format?

It should make processing easier, example:

```
~/work/cli/acceptance % cat requests.txt
{"body": "abc", "headers": {"host": "bla"}}
{"body": "cde", "headers": {}}

% cat requests.txt | jq .body
"abc"
"cde"

% cat requests.txt | jq .headers.host
"bla"
null
```